faucet/src/faucet.rs

```diff
@@ -275,12 +279,12 @@ async fn request_fee_assets(
 
 async fn spendable_records(
     wallet: &CapeWallet<'static, CapeBackend<'static, LoaderMetadata>>,
-    grant_size: u64,
+    grant_size: RecordAmount,
 ) -> impl Iterator<Item = RecordInfo> {
     let now = wallet.lock().await.state().txn_state.validator.now();
     wallet.records().await.filter(move |record| {
         record.ro.asset_def.code == AssetCode::native()
-            && record.ro.amount >= grant_size
+            && record.ro.amount >= grant_size.into()
```

Comment:
You can also do `record.amount()` which returns a `RecordAmount`, then you wouldn't need `.into()`